### PR TITLE
feat: impl ModuleConfigs for standalone

### PR DESCRIPTION
**Pull request by killagu**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: standalone/standalone/test/fixtures/module-with-config/foo.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  hello() {
+    return 'hello from ctx';
+  }
+}
+
+@ContextProto()
+@Runner()
+export class Foo implements MainRunner<object> {
+  @Inject()
+  moduleConfigs: ModuleConfigs;
```
Comment (fengmk2) [inline standalone/standalone/test/fixtures/module-with-config/foo.ts:23]: 如果是当前 module 的配置呢？是否有更简便的方式？

Comment (killagu) [reply standalone/standalone/test/fixtures/module-with-config/foo.ts:23]:
```
@Inject()
moduleConfig: ModuleConfig;
```

这样？

Comment (fengmk2) [reply standalone/standalone/test/fixtures/module-with-config/foo.ts:23]: 可以的

**Review by fengmk2 — APPROVED** [review]
+1

**Pull request merged**